systemc/model_two_power_mod.cpp

```diff
@@ -12,98 +12,95 @@ using namespace sc_dt;
 using namespace verilog;
 
 constexpr int kBW = 256;
+using KeyType = vint<false, 257>;
+struct InType {
```

Comment:
This name sucks, use `RSATwoPowerModIn` ?

Reply:
Fixed.